xtask/bench/benches/libs-graphql.txt

```diff
@@ -0,0 +1,3 @@
+https://raw.githubusercontent.com/amplication/amplication/909a1a77cd52e7f362cb5e037b3928146cef2ae4/schema.graphql
+https://raw.githubusercontent.com/dgraph-io/dgraph/2a3dacd849c912643949a5ae6d78258b6c8f71ab/graphql/e2e/normal/schema.graphql
+https://raw.githubusercontent.com/superfly/flyctl/9255529e9f423e6aca6eba64457187b6aead53fa/gql/schema.graphql
```

Comment:
This one is soo good!

Reply:
It is! I also found the schema of GitHub APIs, but it's 1.1MB ... 😅 It can wait I suppose